app/uk/gov/hmrc/helptosavefrontend/controllers/email/VerifyEmailBehaviour.scala

```diff
@@ -79,7 +79,7 @@ trait VerifyEmailBehaviour { this: HelpToSaveAuth ⇒
       case Failure(e) ⇒
         logger.warn(s"Could not decode email verification parameters: ${e.getMessage}", e)
         val nino = htsContext.nino
-        auditor.sendEvent(SuspiciousActivity(nino, "malformed_redirect"), nino)
+        auditor.sendEvent(SuspiciousActivity(Some(nino), "malformed_redirect"), nino)
```

Comment:
Pls change logger.warn string to "SuspiciousActivity: malformed redirect from email verification service back to HtS, could not decode email verification parameters: ${e.getMessage}"